From 7dfdc0fb62360a671e6e7d2589e6481d1df91af7 Mon Sep 17 00:00:00 2001
From: Po-Hsu Lin <po-hsu.lin@canonical.com>
Date: Tue, 27 Jun 2017 14:17:11 +0800
Subject: [PATCH 36/41] selftests: typo correction for memory-hotplug test

Typo fixed for hotpluggable_offline_memory() in memory-hotplug test.

Signed-off-by: Po-Hsu Lin <po-hsu.lin@canonical.com>
Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>
---
 tools/testing/selftests/memory-hotplug/mem-on-off-test.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh b/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh
index 6cddde0b96f8..a8d858767102 100755
--- a/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh
+++ b/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh
@@ -39,7 +39,7 @@ hotpluggable_memory()
 	done
 }
 
-hotplaggable_offline_memory()
+hotpluggable_offline_memory()
 {
 	hotpluggable_memory offline
 }
@@ -150,7 +150,7 @@ echo -e "\t online all hotplug memory in offline state"
 #
 # Online all hot-pluggable memory
 #
-for memory in `hotplaggable_offline_memory`; do
+for memory in `hotpluggable_offline_memory`; do
 	echo offline-online $memory
 	online_memory_expect_success $memory
 done
@@ -168,7 +168,7 @@ done
 #
 # Online all hot-pluggable memory again
 #
-for memory in `hotplaggable_offline_memory`; do
+for memory in `hotpluggable_offline_memory`; do
 	echo offline-online $memory
 	online_memory_expect_success $memory
 done
@@ -214,7 +214,7 @@ done
 # Test memory hot-add error handling (offline => online)
 #
 echo $error > $NOTIFIER_ERR_INJECT_DIR/actions/MEM_GOING_ONLINE/error
-for memory in `hotplaggable_offline_memory`; do
+for memory in `hotpluggable_offline_memory`; do
 	online_memory_expect_fail $memory
 done
 
@@ -222,7 +222,7 @@ done
 # Online all hot-pluggable memory
 #
 echo 0 > $NOTIFIER_ERR_INJECT_DIR/actions/MEM_GOING_ONLINE/error
-for memory in `hotplaggable_offline_memory`; do
+for memory in `hotpluggable_offline_memory`; do
 	online_memory_expect_success $memory
 done
 
-- 
2.13.0

